src/app/scopes-dialog/scopes-dialog.component.html

```diff
@@ -1,42 +1,50 @@
-<div class="ms-Dialog center-dialog ms-Dialog--close" id="scopes-dialog">
-    <button class="ms-Dialog-button ms-Dialog-buttonClose">
+<dialog id="scopes-dialog" #scopesDialog (keyup.esc)="closeModal()">
```

Comment:
We lost the fabric classes

Reply:
Since we are not using fabric components, the fabric styles will not play nice with this element. I will have to restyle this dialog. 

What do you think?

Reply:
Why won't the fabric styles work with the keyup event? Can we register the keyup event on the button?

I'm all for us using semantic HTML whenever possible. We should try to maintain the look and feel until we have a plan to change it.